src/aat/resources/features/F-044/S-579_Later_Case_Read_By_Solicitor_1.td.json

```diff
@@ -0,0 +1,162 @@
+        },
+        "DocumentField2": {
+          "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943",
+          "document_filename": "Elastic Search test Case.png",
+          "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943/binary"
+        },
+        "ComplexField": {
+          "ComplexTextField": null,
+          "ComplexNestedField": {
+            "NestedNumberField": null,
+            "NestedCollectionTextField": []
+          },
+          "ComplexFixedListField": null
+        },
+        "FixedRadioListField": "RADIOVALUE2",
+        "DateTimeField": null,
+        "PhoneUKField": null,
+        "DocumentField7": {
+          "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/5c14f8ef-887b-41e0-ad01-23959c082311",
+          "document_filename": "Screenshot 2019-12-24 at 3.41.04 pm.png",
+          "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/5c14f8ef-887b-41e0-ad01-23959c082311/binary"
+        },
+        "NumberField": "111111222222",
+        "MultiSelectListField": [
+          "OPTION5",
+          "OPTION4",
+          "OPTION3"
+        ],
+        "YesOrNoField": "Yes",
+        "EmailField": "testemail@gmail.com",
+        "TextField": "sometextfield",
+        "DateField": "2222-01-02",
+        "TextAreaField": "textarea",
+        "CollectionField": []
+      },
+      "data_classification": {
+        "DocumentField4": "PUBLIC",
+        "MoneyGBPField": "PUBLIC",
+        "FixedListField": "PUBLIC",
+        "AddressUKField": {
+          "value": {
+            "County": "PUBLIC",
+            "Country": "PUBLIC",
+            "PostCode": "PUBLIC",
+            "PostTown": "PUBLIC",
+            "AddressLine1": "PUBLIC",
+            "AddressLine2": "PUBLIC",
+            "AddressLine3": "PUBLIC"
+          },
+          "classification": "PUBLIC"
+        },
+        "DocumentField5": "PUBLIC",
+        "DocumentField2": "PUBLIC",
+        "FixedRadioListField": "PUBLIC",
+        "DateTimeField": "PUBLIC",
+        "PhoneUKField": "PUBLIC",
+        "DocumentField7": "PUBLIC",
+        "NumberField": "PUBLIC",
+        "MultiSelectListField": "PUBLIC",
+        "YesOrNoField": "PUBLIC",
+        "EmailField": "PUBLIC",
+        "TextField": "PUBLIC",
+        "DateField": "PUBLIC",
+        "TextAreaField": "PUBLIC",
+        "CollectionField": {
+          "value": [],
+          "classification": "PUBLIC"
+        }
+      },
+      "after_submit_callback_response": null,
+      "callback_response_status_code": null,
+      "callback_response_status": null,
+      "delete_draft_response_status_code": null,
+      "delete_draft_response_status": null,
+      "security_classifications": {
```

Comment:
Can use inheritance.

Reply:
No, we can't use as the data is different